.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Maven Build
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  maven-build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+        java: [1.8, 11, 13]
```

Comment:
@olenagerasimova java `1.8` is not compatible with java 11 specified in `pom.xml`, please fix it